### PR TITLE
[zh] Fix broken link in Service concept page

### PR DESCRIPTION
**Pull request by devhg**
fix

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [zh] Fix broken link in Service concept page

**Comment by windsonsea** [comment]
Hi, this page has another change. Could you please sync it with en too? 

This section is remove, which you can find by running `./scripts/lsync.sh content/zh-cn/docs/concepts/services-networking/service.md`

image

**Review by my-git9 — COMMENTED** [review]
/hold

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -248,7 +248,7 @@ As many Services need to expose more than one port, Kubernetes supports multiple
 port definitions on a Service object.
 Each port definition can have the same `protocol`, or a different one.
 -->
-服务的默认协议是 TCP(/zh-cn/docs/reference/networking/service-protocols/#protocol-tcp)；
+服务的默认协议是[TCP](/zh-cn/docs/reference/networking/service-protocols/#protocol-tcp)；
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/service.md:251]:
```suggestion
服务的默认协议是 [TCP](/zh-cn/docs/reference/networking/service-protocols/#protocol-tcp)；
```

**Comment by tengqm** [comment]
The change looks okay now, but the CLA is failing.
/hold cancel
/easycla

**Comment by devhg** [comment]
> The change looks okay now, but the CLA is failing. /hold cancel /easycla

CLA is ok

**Review by my-git9 — COMMENTED** [review]
/label tide/merge-method-squash
/lgtm

**Pull request merged**